README.md

```diff
@@ -3,15 +3,84 @@
 This repository contains the REST API running as a PubNub function that is required for
 the [ChatEngine Framework](https://github.com/pubnub/chat-engine/) to operate.
 
-## Setup
+## ChatEngine Application and Key Provisioning Guide
 
-First, configure your application in the PubNub administration UI.
+To setup ChatEngine on PubNub, one must first setup a new PubNub App. The following outlines how to manually setup a PubNub ChatEngine Key.
 
-[How to Setup your account for ChatEngine](guides/ChatEngine_Application_and_Key_Provisioning.md)
+Setup an account, create a set up pubsub keys, configure these keys to 
 
-Then, build the function in this repository and upload it.
+### Basic Steps
+1. Navigate to [PubNub's Portal](pubnub.com), sign in, and select the _Create New App_ button at the top-right of the logged in user's app page.
+1. Enter an App name (e.g. `ChatEngine Application`) and click the _Create_ button.
+1. Select the newly made App icon and navigate to the _Demo Keyset_.
+1.  The following setting need to be updated on the key:
 
-[How to Provision your Function](guides/ChatEngine_Server_Provisioning_Guide.md)
+| **Presence**      | `on` |
+|:--------------|:--------:|
+| Announce Max | `20` | 
+| Interval | `10` |
+| Presence Deltas | `Disabled` |
+| Generate Leave on TCP FIN or FIN | `Disabled` |
+| Global Here Now | `Enabled` |
+| Debounce | `2` |
+| Callback State Change | `https://pubsub.pubnub.com/v1/blocks/sub-key/` <br> `__SUB_KEY__/chat-engine-server?route=user_state` |
+
+| **Storage & Playback** | `on` |
+|:--------------|:--------:|
+| Retention | `7 days` |
+
+| **Stream Controllers** | `on` |
+|:--------------|:--------:|
+| Enable Wildcard Subscribe | `Enabled` |
+
+| **PubNub Functions** | `Enabled` |
+|:--------------|:--------:|
+
+| **Access Manager** | `Enabled` |
+|:--------------|:--------:|
+
+## ChatEngine Server Provisioning Guide
```

Comment:
Provisioning ChatEngine Server (and then you can delete the "basic steps" h3, and move everything else up a level)